ArduCopter/mode_planckrtb.cpp

```diff
@@ -22,8 +22,12 @@ bool Copter::ModePlanckRTB::init(bool ignore_checks){
 }
 
 void Copter::ModePlanckRTB::run(){
-    if(!_is_landing)
-    {
+    //Check for a case where we requested the landing but our request was rejected
```

Comment:
Is it an issue that `_is_landing=true` until we get the nack or timeout?
https://github.com/planckaero/ardupilot/blob/d24301434298f950790507907d0a2895d74d2f69/ArduCopter/mode_planckrtb.cpp#L10#L13

Reply:
It actually needs to be true for the retry to be triggered: https://github.com/planckaero/ardupilot/blob/d24301434298f950790507907d0a2895d74d2f69/ArduCopter/mode_planckrtb.cpp#L26

Reply:
I was referring to the fact that it will be `true` for a little bit before is is set to `false`
Since it is set to `true` during the init here: https://github.com/planckaero/ardupilot/blob/d24301434298f950790507907d0a2895d74d2f69/ArduCopter/mode_planckrtb.cpp#L10#L13

Reply:
Understood.  The only place `_is_landing` is used is for that check to see if it got a nack.  The `_is_landing` is used to determine if it should request or re-request the landing.

Reply:
Ok, got it so it's not an issue that is goes to `true` for a little bit before going to false